ISSUE-TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+# Issue name
+
+* Related Issues : _#IssueNumber if necessary_
+
+## Introduction
+
+A short description of what the features or the bugs are. Try to keep it simple and within a
+single-paragraph.
+
+## Issue Description
+
+Describe the bug found or the feature that you are proposing
+
+If they are part of some common pattern hard to express, show
+how one can currently get a similar effect and describe its
+drawbacks. 
+
+If it's completely new feature motivate why this new functionality is necessary, and propose some advice for the development (or open a Pull Request if you have already started to work on the feature on your own)
```

Comment:
The usage of "motive" here doesn't make much sense.  You probably mean "explain" instead.

Also missing a period at the end of this sentence.